toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -15,61 +15,61 @@ rpm_path=$3
 log_path=$4
 
 chroot_name="worker_chroot"
-chroot_builder_folder=${chroot_base}/${chroot_name}
-chroot_archive=${chroot_base}/${chroot_name}.tar.gz
-chroot_log=${log_path}/${chroot_name}.log
+chroot_builder_folder=$chroot_base/$chroot_name
```

Comment:
Just curious - Why rename all these `${variable}`?

Reply:
Just to be closer to what we do in most of our other scripts. It also makes it easier to find places, where we do something more, than just printing the value of the variable. For instance, `${chroot_base:?}` is more visible now in my opinion.